test/system/030-run.bats

```diff
@@ -736,4 +736,14 @@ EOF
     is "$output" "$random_1" "output matches STDIN"
 }
 
+# rhbz#1763007 : the --log-opt for podman run does not work as expected
+@test "podman run with log-opt option" {
+    run_podman run -d --rm --log-opt max-size=1m $IMAGE sleep 5
```

Comment:
It can be useful to test with variable settings. Suggestion:
```bash
    # Pseudorandom size of the form N.NNN. The '| 1' handles '0.NNN' or 'N.NN0',
    # which podman displays as 'NNN kB' or 'N.NN MB' respectively.
    size=$(printf "%d.%03d" $(($RANDOM % 10 | 1)) $(($RANDOM % 100 | 1)))
    run_podman run -d --rm --log-opt max-size=${size}m $IMAGE sleep 5
    cid=$output
    run_podman inspect --format "{{ .HostConfig.LogConfig.Size }}" $cid
    is "$output" "${size}MB"
```